CHANGELOG

```diff
@@ -1,6 +1,14 @@
 devel
 -----
 
+* Allow to specify a fail-over LDAP server. Instead of "--ldap.OPTION" you need
+  to specify "--ldap2.OPTION". Authentication / Authorization will first check
+  the primary LDAP server. If this server cannot authenticate a user, it will
+  try the secondary one. It is possible to specify a file containing all users
+  that the primary (or secondary) LDAP server is handling by specifying the
+  option "--ldap.responsible-for". This file must contain the usernames
+  line-by-line.
```

Comment:
This seems to be an artifact of a merge conflict - the original changelog entry is now missing.

Reply:
thx for paying attention